modules/using-oci-containers-for-model-storage.adoc

```diff
@@ -0,0 +1,207 @@
+
+For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+endif::[]
+
+.Prerequisites
+* You have a model in the ONNX format.
+
+== Creating an OCI image and uploading a model to the container image
+
+.Procedure
+. From your local machine, create a temporary directory to store both the downloaded model and support files to create the OCI image:
++
+[source]
+----
+cd $(mktemp -d)
+----
++
+. Create a `models/` folder inside the temporary directory and download your model:
++
+[source]
+----
+mkdir -p models/1
+
+DOWNLOAD_URL=https://github.com/onnx/models/raw/main/validated/vision/classification/mobilenet/model/mobilenetv2-7.onnx
+curl -L $DOWNLOAD_URL -O --output-dir models/1/
+----
++
+[NOTE]
+====
+The subdirectory `1` is used because OpenVINO requires numbered subdirectories for model versioning. If you are not using OpenVINO, you do not need to create the `1` subdirectory to use OCI container images.
+====
+. Create a Docker file named `Containerfile` with the following contents:
++
+[source]
+----
+FROM registry.access.redhat.com/ubi9/ubi-micro:latest
+COPY --chown=0:0 models /models
+RUN chmod -R a=rX /models
+
+# nobody user
+USER 65534 
+----
++
+[NOTE]
+====
+* In this example, `ubi9-micro` is used as a base container image. You cannot use empty images that do not provide a shell, such as `scratch`, because KServe uses the shell to ensure the model files are accessible to the model server. 
+* Ownership of the copied model files and read permissions are granted to the `root` group. OpenShift runs containers with a random user ID and the `root` group ID. Changing ownership of the group ensures that the model server can access them.
+====
++
+. Confirm that the models follow the directory structure shown using the `tree` command:
++
+[source]
+----
+tree
+
+.
+├── Containerfile
+└── models
+    └── 1
+        └── mobilenetv2-7.onnx
+----
++
+. Create the OCI container image with Podman, and upload it to a registry. For
+example, using Quay as the registry:
++
+[source]
+----
+podman build --format=oci -t quay.io/<user_name>/<repository_name>:<tag_name> .
+podman push quay.io/<user_name>/<repository_name>:<tag_name>
+----
++
+[NOTE]
+====
+If your repository is private, ensure you are authenticated to the registry before uploading your container image.
+====
+
+== Deploying a model stored in an OCI image from a public repository
+
+[NOTE]
+====
+By default in KServe, models are exposed outside the cluster and not protected with authorization. 
+====
+
+. Create a namespace to deploy the model:
++
+[source]
+----
+oc new-project oci-model-example
+----
++
+. Use the OpenData Hub project `kserve-ovms` template to create a `ServingRuntime` resource and configure the OpenVINO model server in the new namespace:
```

Comment:
Do we need to have "OpenData Hub" conditionalized to show RHOAI if on docs.redhat.com? @israel-hdez

Reply:
change to `OpenData Hub`  to  `redhat-ods-applications`

Reply:
So, on docs.redhat.com it should show as "RHOAI Applications project". If these docs also go to https://opendatahub.io/docs, I think we need the conditional.

Reply:
Thanks! I added the conditional back in with your suggestions.